### DISCUSSION TITLE
Demo site access

### DISCUSSION THREAD

**Discussion by dubthree** [Taskana/taskana#1953] [Q&A] [answered]
How do I access your demo site?  It's prompting for a username/password.

Thanks,
Walter

**Comment by cnguyen-de** [comment, marked as answer]
The classical admin admin 👀

**Reply by dubthree** [reply]
Got it!  Thank you.

**Reply by mustaphazorgati** [reply]
If you want to explore our demo as a non-admin you can also login as any of our test users. The password is always the username.
Here you can find our documentation about our current test users: https://taskana.atlassian.net/wiki/spaces/TAS/pages/1150189569/Test+User+Structure

And here you can find the ldap configuration (which is usually more precise than the documentation), where you'll find the test user definitions: https://github.com/Taskana/taskana/blob/master/rest/taskana-rest-spring-example-common/src/main/resources/taskana-example.ldif

**Reply by dubthree** [reply]
Great, thank you.  Did you see my failed attempts to login with user/user?  :)

**Reply by mustaphazorgati** [reply]
I probably could if I'd check the logs. Since it's just a demo environment I didn't check yet.

**Reply by mustaphazorgati** [reply]
Are you interested in using TASKANA? I can offer you a small chat where we can walk through TASKANAs functionality.
If yes: Get in touch with me: mustapha.zorgati@novatec-gmbh.de
